sorachung/pet-journal
# Src nav

```text
#### Changes Made
1. Deleted file `Sidebar.js` from `navbar` directory.
2. Modified file `Navbar.js` to include `preliminary sidebar and navbar` and functionality using MUI.
​

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-nav
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Start app
    > log in as a user
    > open sidebar, try links in sidebar
    > try clicking profile icon and pet icon to see if menu opens
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
